src/cylinder.rs

```diff
@@ -18,10 +18,16 @@
 use cgmath::Point3;
 use cgmath::Vector3;
 
-#[derive(Copy, Clone, PartialEq)]
+/// Bounding cylinder
+#[derive(Copy, Clone, PartialEq, Debug)]
 #[cfg_attr(feature = "eders", derive(Serialize, Deserialize))]
 pub struct Cylinder<S> {
+    /// Center point
     pub center: Point3<S>,
+
```

Comment:
not sure why we'd need empty lines here

Reply:
It's just one of my quirks, I find it easier to read the code that way. If you want me to remove them I can do that.